src/main/java/no/unit/nva/bare/AddNewAuthorityIdentifierHandler.java

```diff
@@ -97,11 +99,12 @@ private void validateInput(AddNewAuthorityIdentifierRequest input)
         }
     }
 
-    protected Authority addNewIdentifier(String scn, String qualifier, String identifier)
+
+    protected Authority addNewIdentifier(String scn, AuthorityIdentifier authorityIdentifier)
             throws ApiGatewayException {
         try {
-            HttpResponse<String> response = bareConnection.addNewIdentifier(scn, qualifier, identifier);
-            if (response.statusCode() == SC_OK) {
+            HttpResponse<String> response = bareConnection.addIdentifier(scn, authorityIdentifier);
```

Comment:
No content in case you request it more than once?

Reply:
Yes, trying to be idempotent

Reply:
The bug was initially that the wrong method was called in the used authority-api - it could not handle URIs on the rest path